### PR TITLE
Start dbus for Chrome

### PR DESCRIPTION
**Pull request by OrKoN**
This PR adds a command to start dbus. As discovered in #42220 Chrome prints many errors that it is not able to access dbus and one of the reasons might be that the dbus service is not started in CI.

### CODE REVIEW

**Comment by OrKoN** [comment]
Unfortunately, I could not find where to look for the Azue Pipelines errors and I'd appreciate if anyone could extract a direct link.

File: tools/docker/Dockerfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ RUN apt-get -qqy update \
     bzip2 \
```
Comment (jgraham) [inline tools/docker/Dockerfile:11]: Unfortunately you can't change the docker file that easily; you'd need to update the version number and arrange for a manual push. However the fact that this is working suggests maybe dbus is already installed?

Comment (OrKoN) [reply tools/docker/Dockerfile:11]: Oh, so the changed dockerfile would not be used in the CI run? Do you know if it's possible to trigger wpt.fyi run for this PR using the updated Dockerfile? I want to test if this change makes the difference for the errors described in https://github.com/web-platform-tests/wpt/issues/42220.

**Review by thiagowfx — APPROVED** [review]

**Comment by whimboo** [comment]
> With this change the previous error went away, now there is a different dbus related error:
> 
> ```
>  8:50.08 pid:8029 [9673:9695:1005/074511.871189:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
> ```

Does that still cause slowness of Chrome as it can be seen on https://github.com/web-platform-tests/wpt/issues/42220?

**Comment by OrKoN** [comment]
@whimboo it still times out but I am not 100% sure it is caused by dbus (locally it does not time out but it might that bots are just slower).

**Review by thiagowfx — APPROVED** [review]

**Review by DanielRyanSmith — APPROVED** [review]

**Pull request merged**